test/integration/proxy-test.js

```diff
@@ -978,7 +978,8 @@ test('handle tchannel failures', function t(assert) {
         setTimeout(function onTimer() {
             var name = cluster.two.channel.hostPort;
             var peer = cluster.one.channel.peers.get(name);
-            var conn = peer.connections[0]; // the "out" one
+
```

Comment:
This is part of the "listen on top channel only" fix necessary; see seperate commit.